chia/_tests/core/mempool/test_singleton_fast_forward.py

```diff
@@ -271,10 +270,9 @@ async def make_and_send_spend_bundle(
 
 async def get_singleton_and_remaining_coins(sim: SpendSim) -> Tuple[Coin, List[Coin]]:
     coins = await sim.all_non_reward_coins()
-    singletons, remaining_coins = partition(lambda coin: coin.amount % 2 == 0, coins)
-    singletons_list = list(singletons)
-    assert len(singletons_list) == 1
```

Comment:
you don't have a counterpart to this assert now. if there are two singletons in the list, that would previously be an error. in your version it silently returns the remaining singletons as part of `remaining_coins`

Reply:
The whole setup is centered around one singleton, but why not. Added.